src/signals/incident-management/containers/IncidentSplitContainer/components/SplitForm/index.js

```diff
@@ -97,40 +130,54 @@ class SplitForm extends React.Component {
           attachments={attachments}
           subcategories={subcategories}
           priorityList={priorityList}
-          splitForm={this.state.splitForm}
+          splitForm={this.form}
         />
 
         {this.state.isVisible ?
-          <div>
-            <button onClick={() => this.setVisibility(false)} className="action reset split-form__button-hide">Verwijder</button>
+          <Fragment>
+            <StyledRemoveButton
+              data-testid="splitForPartRemove"
+              variant="textButton"
+              onClick={() => this.setVisibility(false)}
+            >Verwijder</StyledRemoveButton>
 
             <IncidentPart
               index="3"
               incident={incident}
               attachments={attachments}
               subcategories={subcategories}
               priorityList={priorityList}
-              splitForm={this.state.splitForm}
+              splitForm={this.form}
             />
 
-          </div>
+          </Fragment>
           :
-          <button onClick={() => this.setVisibility(true)} className="action tertiair split-form__button-show">Deelmelding 3 toevoegen</button>
+          <StyledButton
+            data-testid="splitForPartAdd"
+            variant="primaryInverted"
+            onClick={() => this.setVisibility(true)}
+          >Deelmelding 3 toevoegen</StyledButton>
         }
 
-        <div className="split-form__disclainer">
-          <h4>Let op</h4>
+        <StyledBottomDisclaimer data-testid="splitFormBottomDisclaimer">
+          <StyledH4 $as="h4">Let op</StyledH4>
           <ul>
             <li>De persoon die de oorspronkelijke melding heeft gedaan, ontvangt een email per deelmelding.</li>
             <li>De oorspronkelijke melding wordt afgesloten als deze gesplitst wordt.</li>
             <li>Een melding kan maar 1 keer gesplitst worden.</li>
           </ul>
-        </div>
-
-        <div>
-          <button onClick={this.handleSubmit} className="action primary">Splitsen</button>
-          <button onClick={handleCancel} className="action tertiair">Annuleer</button>
-        </div>
+        </StyledBottomDisclaimer>
+
+        <Fragment>
```

Comment:
Is a `<Fragment>` component needed in this case?

Reply:
scherp gedaan